lib/constants/styles.dart

```diff
@@ -15,3 +15,38 @@ const kSettingsTextW700 = TextStyle(fontSize: 16, fontWeight: FontWeight.w700);
 
 const kSettingsTextW400 = TextStyle(
     fontSize: 16, fontWeight: FontWeight.w400, color: kGrayScaleLightest,);
+
+class HyhTextStyle {
+
+  //Grayscale Lightest styles
+  static const body14 = TextStyle(fontSize: 14, fontWeight: FontWeight.w400, color: kGrayScaleLightest);
+  static const body14Bold = TextStyle(fontSize: 14, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+  static const body16 = TextStyle(fontSize: 16, fontWeight: FontWeight.w400, color: kGrayScaleLightest);
+  static const body16Height15 = TextStyle(fontSize: 16, fontWeight: FontWeight.w400, color: kGrayScaleLightest, height: 1.5);
+  static const body16Bold = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+  static const body16Height15Bold = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kGrayScaleLightest, height: 1.5);
+  static const heading18 = TextStyle(fontSize: 18, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+  static const heading32 = TextStyle(fontSize: 32, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+
+  //Grayscale Light styles
+  static const heading32Light = TextStyle(fontSize: 32, fontWeight: FontWeight.w700, color: kGrayScaleLight);
```

Comment:
Is it expected that `heading32Light` says light but the weight is `w700`?

Reply:
i thought just in the color, but maybe should rename to lightGray